modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -64,44 +65,44 @@ func (a TransferAuthorization) Accept(ctx sdk.Context, msg sdk.Msg) (authz.Accep
 			}}, nil
 		}
 	}
-	return authz.AcceptResponse{}, sdkerrors.Wrapf(sdkerrors.ErrNotFound, "requested port and channel allocation does not exist")
+	return authz.AcceptResponse{}, errorsmod.Wrapf(ibcerrors.ErrNotFound, "requested port and channel allocation does not exist")
 }
 
 // ValidateBasic implements Authorization.ValidateBasic.
 func (a TransferAuthorization) ValidateBasic() error {
 	if len(a.Allocations) == 0 {
-		return sdkerrors.Wrap(ErrInvalidAuthorization, "allocations cannot be empty")
+		return errorsmod.Wrap(ErrInvalidAuthorization, "allocations cannot be empty")
 	}
 
 	foundChannels := make(map[string]bool, 0)
 
 	for _, allocation := range a.Allocations {
 		if _, found := foundChannels[allocation.SourceChannel]; found {
-			return sdkerrors.Wrapf(channeltypes.ErrInvalidChannel, "duplicate source channel ID: %s", allocation.SourceChannel)
+			return errorsmod.Wrapf(channeltypes.ErrInvalidChannel, "duplicate source channel ID: %s", allocation.SourceChannel)
 		}
 
 		foundChannels[allocation.SourceChannel] = true
 
 		if allocation.SpendLimit == nil {
-			return sdkerrors.Wrap(sdkerrors.ErrInvalidCoins, "spend limit cannot be nil")
+			return errorsmod.Wrap(ibcerrors.ErrInvalidCoins, "spend limit cannot be nil")
 		}
 
 		if err := allocation.SpendLimit.Validate(); err != nil {
-			return sdkerrors.Wrapf(sdkerrors.ErrInvalidCoins, err.Error())
+			return errorsmod.Wrapf(ibcerrors.ErrInvalidCoins, err.Error())
 		}
 
 		if err := host.PortIdentifierValidator(allocation.SourcePort); err != nil {
-			return sdkerrors.Wrap(err, "invalid source port ID")
+			return errorsmod.Wrap(err, "invalid source port ID")
 		}
 
 		if err := host.ChannelIdentifierValidator(allocation.SourceChannel); err != nil {
-			return sdkerrors.Wrap(err, "invalid source channel ID")
+			return errorsmod.Wrap(err, "invalid source channel ID")
 		}
 
 		found := make(map[string]bool, 0)
 		for i := 0; i < len(allocation.AllowList); i++ {
 			if found[allocation.AllowList[i]] {
-				return sdkerrors.Wrapf(ErrInvalidAuthorization, "duplicate entry in allow list %s")
+				return errorsmod.Wrapf(ErrInvalidAuthorization, "duplicate entry in allow list %s", allocation.AllowList[i])
```

Comment:
Wrapf was being used but no arg was being passed. I've now passed the duplicate entry.